contracts/governance/Staking/StakingStorage.sol

```diff
@@ -149,4 +148,20 @@ contract StakingStorage is Ownable {
 
     /// @dev Staking contract is frozen
     bool public frozen;
+
+    /// @dev max iterations that can be supported in 1 tx for the withdrawal
+    uint256 internal maxVestingWithdrawIterations;
+
+    /// @dev mapping (duration => creationType). Duration is the vesting duration.
+    mapping(uint256 => uint256) public vestingCreationType;
```

Comment:
delete

Reply:
deleted in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/19ef0dedeffdf80545269ffafe2f1af684a25a06)